_howdoi/install-oshinko-cli.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+[id='install-oshinko-cli']
+= install the Oshinko command line interface tool
```

Comment:
wondering if how to install oshinko cli should be the first how-to that people see

Reply:
++

we have an open issue (#218) that speaks to a need for ordering on these things. i haven't come across a good solution to the problem.

i _think_ some sort of categorized sections would be good, but i'm not sure _what_ those categories should be. this is complicated by articles that straddle 2 boundaries. totally open to suggestions here.